app/models/planet.py

```diff
@@ -0,0 +1,29 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String, nullable=False)
+    description = db.Column(db.String, nullable=False)
+    gravity = db.Column(db.Float, nullable=False)
+
+    def to_dict(self):
+        planet_dict = dict(
+            id = self.id,
+            name = self.name,
+            description = self.description,
+            gravity = self.gravity
+        )
+        return planet_dict
```

Comment:
This is perfectly valid, but we could also return the dictionary without storing it in a variable like in `from_dict` below.